keyboards/thevankeyboards/minivan/keymaps/king/keymap.c

```diff
@@ -0,0 +1,116 @@
+#include QMK_KEYBOARD_H
+extern keymap_config_t keymap_config;
+
+
+enum layer_names {
+    _ML,
+    _FL,
+    _NL,
+    _SL,
+    _RL
+};
+
+
+#define ______ KC_TRNS
```

Comment:
```suggestion
```

If you add one more `_` to these below, you won't need this as it's already defined in QMK core.